sonar-docs/analysis/scan/sonarscanner-for-msbuild.md

```diff
@@ -0,0 +1,203 @@
+  <!-- Mark the project as being a test project -->
+  <SonarQubeTestProject>true</SonarQubeTestProject>
+</PropertyGroup>
+```
+
+**Per-project analysis parameters**
+Some analysis parameters can be set for a single MSBuild project by adding them to its .csproj file.
+```
+<!-- in .csproj -->
+<ItemGroup>
+  <SonarQubeSetting Include="sonar.stylecop.projectFilePath">
+    <Value>$(MSBuildProjectFullPath)</Value>
+  </SonarQubeSetting>
+</ItemGroup>
+```
+
+**Concurrent Analyses on the Same Build Machine**  
+Concurrent analyses (i.e. parallel analysis of two solutions on the same build machine using a unique service account) are not supported by default by the Scanner for MSBuild. You can enable it as follows:
+
+1. Locate the folder containing the Scanner for MSBuild
+1. Go in the `Targets` folder and copy the folder `SonarQube.Integration.ImportBefore.targets`
+1. Paste it under your build tool global `ImportBefore` folder (if the folder doesn't exist, create it).
+   * For MSBuild, the path is `<MSBUILD_INSTALL_DIR>\<Version>\Microsoft.Common.targets\ImportBefore` where <MSBUILD_INSTALL_DIR> is:
+      *For v14, default path is: `C:\Program Files (x86)\MSBuild\14.0\Microsoft.Common.Targets\ImportBefore`
+      * For v15, default path is: `C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\MSBuild\15.0\Microsoft.Common.targets\ImportBefore` (for VS Community Edition)
+      * For v16, default path is: 'C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Current\Microsoft.Common.targets` (for VS Community Edition)
+   * For dotnet, the path is `<DOTNET_SDK_INSTALL_DIR>\15.0\Microsoft.Common.targets\ImportBefore` where `<DOTNET_SDK_INSTALL_DIR>` can be found using the `dotnet --info` and looking for the Base Path property.
+
+The performance impact of this global installation for projects that aren't analyzed is negligible as this target is only a bootstrapper and will bail out nearly instantaneously when the `.sonarqube` folder is not found under the folder being built.
+
+**Using SonarScanner for MSBuild with a Proxy**  
+On build machines that connect to the Internet through a proxy server you might experience difficulties connecting to {instance}. To instruct the Java VM to use the system proxy settings, you need to set the following environment variable before running the SonarScanner for MSBuild:
+```
+SONAR_SCANNER_OPTS = "-Djava.net.useSystemProxies=true"
+```
+To instruct the Java VM to use specific proxy settings or when there is no system-wide configuration use the following value:
+```
+SONAR_SCANNER_OPTS = "-Dhttp.proxyHost=yourProxyHost -Dhttp.proxyPort=yourProxyPort"
+```
+Where yourProxyHost and yourProxyPort are the hostname and the port of your proxy server. There are additional proxy settings for https, authentication and exclusions that could be passed to the Java VM, for full reference visit the following article: https://docs.oracle.com/javase/8/docs/technotes/guides/net/proxies.html
```

Comment:
It would be better if `yourProxyHost` and `yourProxyPort` were formatted in some way. They are in italics in the current documentation.